doc/rst/source/basemap_common.rst_

```diff
@@ -43,26 +41,26 @@ Optional Arguments
-    map scale (**-L**) or map rose (**-T**) using
-    :term:`MAP_FRAME_PEN`; specify a different pen with **+p**\ *pen*.
-    Add **+g**\ *fill* to fill the box [no fill].
-    Append **+c**\ *clearance* where *clearance* is either *gap*, *xgap*\ /\ *ygap*,
-    or *lgap*\ /\ *rgap*\ /\ *bgap*\ /\ *tgap* where these items are uniform, separate in
-    x- and y-direction, or individual side spacings between map inset/scale/rose and border.
-    Append **+i** to draw a secondary, inner border as well. We use a uniform
-    *gap* between borders of 2\ **p** and the :term:`MAP_DEFAULT_PEN`
-    unless other values are specified. Append **+r** to draw rounded
-    rectangular borders instead, with a 6\ **p** corner radius. You can
-    override this radius by appending another value. Finally, append
-    **+s** to draw an offset background shaded region. Here, *dx*/*dy*
-    indicates the shift relative to the foreground frame
-    [4\ **p**/-4\ **p**] and *shade* sets the fill style to use for shading [gray50].
-    Used in combination with **-D**, **-L** or **-T**. To specify separate parameters
-    for the various map features, append  **d**\|\ **l**\|\ **t** to **-F**
-    to specify panel parameters for just that panel [Default uses the same panel
-    parameters for all selected map features].
+**-F**\ [**d**\|\ **l**\|\ **t**][**+c**\ *clearances*][**+g**\ *fill*][**+i**\ [[*gap*/]\ *pen*]][**+p**\ [*pen*]]\
+[**+r**\ [*radius*]][**+s**\ [[*dx*/*dy*/][*shade*]]]
+
+    Without further options, draws a rectangular border around any map inset (|-D|; classic mode only), map scale
+    (|-L|) or map rose (|-T|) using :term:`MAP_FRAME_PEN`. Used in combination with |-D|, |-L| or |-T|.  To
+    specify separate parameters for the various map features, append  **d**\|\ **l**\|\ **t** to |-F| to specify panel
+    parameters for just that panel [default uses the same panel parameters for all selected map features].
+    The following modifiers can be appended to |-F|:
+
+    * **+p**\ *pen* to specify a different pen width
+    * **+g**\ *fill* to fill the box [default is no fill]
+    * **+c**\ *clearance* where *clearance* is either *gap*, *xgap*\ /\ *ygap*, or *lgap*\ /\ *rgap*\ /\ *bgap*\ /\
+      *tgap* where these items are uniform, separate in x- and y-direction, or individual side spacings between map
+      inset/scale/rose and border respectively.
+    * **+i** to draw a secondary, inner border as well. We use a uniform *gap* between borders of 2\ **p** and the
+      :term:`MAP_DEFAULT_PEN` unless other values are specified.
+    * **+r** to draw rounded rectangular borders instead, with a 6\ **p** corner radius. You can override this radius by
+      appending another value.
+    * **+s** to draw an offset background shaded region. Here, *dx*/*dy* indicates the shift relative to the foreground
+      frame [default is 4\ **p**/-4\ **p**] and *shade* sets the fill style to use for shading [default is gray50].
```

Comment:
Just a quick comment. Personally, I prefer to use `-` (not `*`) for bullet lists, because `*` is heavily used for italic and bold. 
```suggestion
    - **+p**\ *pen* to specify a different pen width
    - **+g**\ *fill* to fill the box [default is no fill]
    - **+c**\ *clearance* where *clearance* is either *gap*, *xgap*\ /\ *ygap*, or *lgap*\ /\ *rgap*\ /\ *bgap*\ /\
      *tgap* where these items are uniform, separate in x- and y-direction, or individual side spacings between map
      inset/scale/rose and border respectively.
    - **+i** to draw a secondary, inner border as well. We use a uniform *gap* between borders of 2\ **p** and the
      :term:`MAP_DEFAULT_PEN` unless other values are specified.
    - **+r** to draw rounded rectangular borders instead, with a 6\ **p** corner radius. You can override this radius by
      appending another value.
    - **+s** to draw an offset background shaded region. Here, *dx*/*dy* indicates the shift relative to the foreground
      frame [default is 4\ **p**/-4\ **p**] and *shade* sets the fill style to use for shading [default is gray50].
```